src/infra/guild-command-manager.ts

```diff
@@ -21,17 +21,32 @@ export class GuildCommandManager {
         try {
             this.finalized = true;
             const rest = new REST({ version: "10" }).setToken(token);
-            M.log("Sending guild commands");
-            await rest.put(
-                Routes.applicationCommands(this.wheatley.id),
-                { body: this.commands },
-            );
-            // Clear any previous guild commands
-            await rest.put(
-                Routes.applicationGuildCommands(this.wheatley.id, TCCPP_ID),
-                { body: [] },
-            );
-            M.log("Finished sending guild commands");
+            if (!this.wheatley.freestanding) {
+                M.log("Sending application commands");
+                await rest.put(
+                    Routes.applicationCommands(this.wheatley.id),
+                    { body: this.commands },
+                );
+                // Clear any previous guild commands
+                await rest.put(
+                    Routes.applicationGuildCommands(this.wheatley.id, this.wheatley.guildId),
+                    { body: [] },
+                );
+            }
+            else {
```

Comment:
Minor, but to match the style of the rest of the codebase can we update this to be
```ts
if(...) {
    ...
} else {
    ...
}
```
(if the if-else exists after the above comment)